README.md

```diff
@@ -6,10 +6,17 @@
 
 ```groovy
    repositories {
-       maven()
+      maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+        maven { url "https://dl.bintray.com/ethereum/maven" }
    }
 
-   testCompile "org.web3j:web3j-unit:4.6.0-SNAPSHOT"
+   testImplementation "org.web3j:web3j-unit:4.6.0-SNAPSHOT"
+```
+or if JUnit already exists as a dependency:
+```
+testImplementation ("org.web3j:web3j-unit:4.6.0-SNAPSHOT"){
+      exclude group:'org.junit.jupiter'
```

Comment:
Can you please try this with a junit 5 project you probably don't need to exclude this. I don't think this will work with junit 4.

Reply:
with JUnit 4 it won't be a problem but if for this is included as a dependency for web3j-cli which uses JUnit 5 then there you can get some weird behavior.

Reply:
ok - I prefer we split the repo and dependency section

1. Add repos to `gradle.build`

repository {
    ...
}

2. Import web3j-unit:
    Junit 5:
    ```
         testImplementation "org.web3j:web3j-unit:4.6.0-SNAPSHOT"
    ```
    Junit 4:
    ```
        testImplementation ("org.web3j:web3j-unit:4.6.0-SNAPSHOT"){
          exclude group:'org.junit.jupiter'
        }
    ```